codespell_lib/data/dictionary.txt

```diff
@@ -19252,6 +19264,7 @@ obession->obsession
 obessions->obsessions
 obgect->object
 obgects->objects
+obhect->object
```

Comment:
The plural one too please.

Reply:
added plural.  Would you also want -ting or -tify or -tyfying versions of this one?

Reply:
added them, can remove if desired.

Reply:
The more the merrier, as you've already found using it.